Detections/LAQueryLogs/UserSearchingForVIPUserActivity.yaml

```diff
@@ -26,24 +26,25 @@ query: |
   | where QueryText has_any (vips) or QueryText has_any ('_GetWatchlist("VIPUsers")', "_GetWatchlist('VIPUsers')")
   | where AADEmail !in (allowed_users)
   | project TimeGenerated, AADEmail, RequestClientApp, QueryText, ResponseRowCount, RequestTarget
-  | extend timestamp = TimeGenerated, AccountCustomEntity = AADEmail
-
+  | extend timestamp = TimeGenerated, AccountName = tostring(split(AADEmail, "@")[0]), AccountUPNSuffix = tostring(split(AADEmail, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
+        columnName: AccountName
+      - identifier: UPNSuffix
+        columnName: AccountUPNSuffix
   - entityType: AzureResource
     fieldMappings:
       - identifier: ResourceId
         columnName: RequestTarget
```

Comment:
Existing change looks good. I noticed mapping for resourceId and was wondering should it be mapped to _ResourceId column ? we may have to project it in that case.

Reply:
We will leave as is to make sure the intent of the query is maintained and no assumptions are made by fixing the mapping.

https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/laquerylogs

<html>
<body>
<!--StartFragment-->

RequestTarget | string | ResourceId of the request URL.
-- | -- | --
_ResourceId | string | A unique identifier for the resource that the record is associated with

<!--EndFragment-->
</body>
</html>